iotdb-core/ainode/ainode/core/manager/model_manager.py

```diff
@@ -112,6 +252,68 @@ def get_ckpt_path(self, model_id: str) -> str:
         """
         return self.model_storage.get_ckpt_path(model_id)
 
+    def _validate_model_name(self, model_name: str) -> bool:
+        """
+        Validate the model name format
+
+        Args:
+            model_name: Name of the model
+
+        Returns:
+            Whether the name is valid
+        """
+        if not model_name:
+            return False
+
+        import re
+
+        pattern = r"^[a-zA-Z0-9_-]+$"
+
+        if re.match(pattern, model_name):
+            logger.debug(f"Model name validated: {model_name}")
+            return True
+        else:
+            logger.error(f"Illegal model name: {model_name}")
+            return False
```

Comment:
We don't need to verify the legality of the model name, cuz the SQL parser in IoTDB has already done this. Maybe we should add this feature in the furture, but not today. Thus, this interface should be removed.